projects/plugins/jetpack/extensions/blocks/anchor-fm/anchor-fm.php

```diff
@@ -141,6 +141,21 @@ function process_anchor_params() {
 							),
 						);
 					}
+				} else {
+					$retry_url         = add_query_arg(
+						array(
+							'anchor_podcast' => $podcast_id,
+							'anchor_episode' => $episode_id,
+							'spotify_url'    => $spotify_url,
+						),
+						admin_url( 'post-new.php' )
+					);
+					$data['actions'][] = array(
+						'create-episode-error-notice',
+						array(
+							'retry_url' => $retry_url,
```

Comment:
Thanks for catching this! It looks like using `esc_url` here generates a different url, since the ampersands are encoded. Is this a valid use case for `esc_url_raw`? Or should we escape each part of the URL?

Reply:
Actually it looks like the url is being escaped twice, and the second time removes the & from `&#038;`. 🤔

Reply:
That's interesting. What happens if you `rawurlencode` the spotify URL as you add it to the query args?

Reply:
I tried `rawurlencode`-ing the Spotify URL and `esc-url`-ing `$retry_url`, but the first ampersand was still being removed from the retry URL. 😕 

Escaped URL | Redirected to
------------- | -------------
`anchor_podcast=22b6608&#038;` | `anchor_podcast=22b6608#038;`

I've [pushed an update](https://github.com/Automattic/jetpack/pull/18573/commits/8a64e47fe66b140a15115b665cc14835931a7344) to `rawurlencode` the Spotify URL but use `esc_url_raw` to avoid encoding the ampersand. Let me know if you agree with that approach.